src/js/utils/helpers.js

```diff
@@ -4,7 +4,7 @@ export const cloneDeep = val => {
   if (val instanceof Array) {
     return val.map(cloneDeep);
   } else if (val instanceof Object) {
-    return Object.keys(val).reduce(
+    return [...Object.getOwnPropertySymbols(val), ...Object.keys(val)].reduce(
```

Comment:
Could this use `Object.entries` & `Object.fromEntries`?

Reply:
Not with symbols.

Reply:
Sadly – a better thing would probably be to import lodash.cloneDeep.